lib/travis/build/script/julia.rb

```diff
@@ -68,21 +68,19 @@ def script
           sh.echo 'Package name determined from repository url to be ${JL_PKG}',
             ansi: :green
           # Check if the repository is a Julia package.
-          sh.if "-f src/${JL_PKG}.jl" do
+          sh.if "-f Project.toml || -f JuliaProject.toml" do
             sh.if '-a .git/shallow' do
               sh.cmd 'git fetch --unshallow'
             end
-            sh.cmd 'julia --color=yes -e "if VERSION < v\"0.7.0-DEV.5183\" || (!isfile(\"Project.toml\") && !isfile(\"JuliaProject.toml\")); Pkg.clone(pwd()); end"'
-            sh.cmd 'julia --color=yes -e "Pkg.build(\"${JL_PKG}\")"'
-            sh.if '-f test/runtests.jl' do
-              sh.cmd 'julia --check-bounds=yes --color=yes ' \
-                '-e "Pkg.test(\"${JL_PKG}\", coverage=true)"'
-            end
+            sh.cmd 'julia --color=yes -e "if VERSION < v\"0.7.0-DEV.5183\"; Pkg.clone(pwd()); Pkg.build(\"${JL_PKG}\"); else using Pkg; Pkg.build(); end"'
```

Comment:
I think it would be good to start adding comments explaining what this is doing, since we're likely to forget. E.g. this line is "Clone package into current directory for old Pkg2 projects"